chaoscenter/authentication/api/handlers/rest/dex_auth_handler.go

```diff
@@ -138,7 +138,20 @@ func DexCallback(userService services.ApplicationService) gin.HandlerFunc {
 			c.JSON(utils.ErrorStatusCodes[utils.ErrServerError], presenter.CreateErrorResponse(utils.ErrServerError))
 			return
 		}
-		jwtToken, err := userService.GetSignedJWT(signedInUser)
+
+		salt, err := userService.GetConfig("salt")
```

Comment:
The salt value being returned here is an encrypted value and should be decrypted before using it to generate the JWT